### PR TITLE
Fix compilation warnings

### PR DESCRIPTION
**Pull request by Geod24**
This fixes a bunch of warning I got while porting it to use dub (using 2.065.0).

### CODE REVIEW

File: ylink.d [hunk 1]
```diff
@@ -45,9 +45,11 @@ void main(string[] args)
             {
             default:
                 args[i] = args[i].defaultExtension("obj");
+		goto case;
```
Comment (yebblies) [inline ylink.d]: Indentation is wrong and should use spaces, not tabs.

**Comment by yebblies** [comment]
I'm not a big fan of enabling warnings in code that is under development, but these seem ok.

**Comment by Geod24** [comment]
The indent should be okay by now.

**Comment by yebblies** [comment]
Thanks!

**Pull request merged**